examples/hal/quad/main.rs

```diff
@@ -314,7 +314,7 @@ fn main() {
         memory_types.iter().find(|mem_type| {
             buffer_req.type_mask & (1 << mem_type.id) != 0 &&
             mem_type.properties.contains(m::CPU_VISIBLE) &&
-            mem_type.properties.contains(m::WRITE_COMBINED)
+            mem_type.properties.contains(m::COHERENT)
```

Comment:
AFAIK, technically COHERENT doesn't imply either upload/download types. It's orthogonal, and thus we shouldn't request it here.

Reply:
Oh yeah, well, we don't expose invalidation and flushing, so COHERENT is a bit useless in our API imo /: